src/figureprinter/FigurePainter.java

```diff
@@ -29,9 +36,18 @@ public static void drawLeftUpTriangle(int length){
      *     * * * * *
      * @param length  specified length of triangle will be drawn
      */
-    public static void drawRightUpTriangle(int length){
-        // TODO : complete the method
 
+
+    public static void drawRightUpTriangle(int length){
+        for (int i = 0; i < length; i++) {
+            for (int j = i; j < length; j++) {
+                System.out.print(" ");
+            }
+            for (int j = 0; j < i; j++) {
```

Comment:
To have four "*" marks at the bottom replace this line   for (int j = 0; j < i; j++) with the following  for (int j = 0; j <= i; j++) {